### PR TITLE
More appropriate wording for background selection

### PR DESCRIPTION
**Pull request by Jerome-Herbinet**
Signed-off-by: Jérôme Herbinet <33763786+Jerome-Herbinet@users.noreply.github.com>

Signed-off-by: Jérôme Herbinet <33763786+Jerome-Herbinet@users.noreply.github.com>

### CODE REVIEW

**Comment by szaimen** [comment]
cc @skjnldsv

**Comment by Jerome-Herbinet** [comment]
> cc @skjnldsv

@szaimen will I need to do a npm run sass in this context ?

**Comment by szaimen** [comment]
> @szaimen will I need to do a npm run build in this context ?

yes

**Comment by Jerome-Herbinet** [comment]
> > @szaimen will I need to do a npm run build in this context ?
> 
> yes

Done ! :-)

**Comment by Jerome-Herbinet** [comment]
Should fix [#5142](https://github.com/CollaboraOnline/online/issues/5142)

**Comment by szaimen** [comment]
@Jerome-Herbinet We should probably do all of the improvements in one go: Could you check out https://github.com/nextcloud/server/issues/34511?

**Comment by Jerome-Herbinet** [comment]
Should fix #3365

**Comment by Jerome-Herbinet** [comment]
> @Jerome-Herbinet We should probably do all of the improvements in one go: Could you check out #34511?

What is the method ?

**Comment by szaimen** [comment]
> What is the method ?

The question was if you could adjust the other places that need to be improved too...

**Comment by Jerome-Herbinet** [comment]
> > What is the method ?
> 
> The question was if you could adjust the other places that need to be improved too...

I'll see how to add my changes to that branch.

File: apps/theming/src/components/BackgroundSettings.vue [hunk 1]
```diff
@@ -208,7 +208,7 @@ export default {
 		},
 
 		pickFile() {
-			window.OC.dialogs.filepicker(t('theming', 'Insert from {productName}', { productName: OC.theme.name }), (path, type) => {
+			window.OC.dialogs.filepicker(t('theming', 'Select a background from your files', { productName: OC.theme.name }), (path, type) => {
```
Comment (skjnldsv) [inline apps/theming/src/components/BackgroundSettings.vue:211]:
```suggestion
			window.OC.dialogs.filepicker(t('theming', 'Select a background from your files'), (path, type) => {
```

Comment (skjnldsv) [inline apps/theming/src/components/BackgroundSettings.vue:211]:
```suggestion
			window.OC.dialogs.filepicker(t('theming', 'Select a background from your files'), (path, type) => {
```

**Review by skjnldsv — CHANGES_REQUESTED** [review]

**Comment by Jerome-Herbinet** [comment]
@szaimen I've been trying to understand exactly what I'm supposed to try/do in relation with the [#34511](https://github.com/nextcloud/server/issues/34511) issue ... and, I'm sorry, but I need more information about exactly what I'm supposed to do now in this context. 🙏🏼

**Comment by Jerome-Herbinet** [comment]
Other problem, I can't resolve the Node / node (pull_request) problem ; can you give me a tip ?

**Review by skjnldsv — APPROVED** [review]

**Comment by szaimen** [comment]
> Do you suggest me to do a rebase of "master" ?

Yes :)

**Comment by Jerome-Herbinet** [comment]
Hi, 
I tried to rebase from my branch to master, but I had a problem with the 3rdparty folder (submodule).
Please check out the output of my try :

```bash
jh-arawa@administrateur-ThinkBook-15-G2-ITL:~/Documents/GitHub/server$ git rebase master 
Rembobinage préalable de head pour pouvoir rejouer votre travail par-dessus...
Génération des patchs: 100% (1423/1423), fait.
Application de  Make chunkperf.php work again
Application de  Remove chunkperf.php
Application de  Port share by mail settings to vue
Application de  [tx-robot] updated from transifex
Application de  Fix crop condition
Application de  Add explicit return if getSmallImagePreview fails
Application de  Use custom PHP runtime for PHPUnit execution in autotest
Application de  Fix spelling for PreConditionNotMetException
Application de  Improve two factor admin settings
Application de  Make appName of TemplateResponse accessible in BeforeTemplateRenderedEvent
Application de  Port admin federated files sharing to vue
Application de  Rename userName to username to align with child classes
Application de  Fix return type for createSpecificUser
Application de  Cleanup admin delegation setting page
Application de  Port BackgroundJob admin settings to vue
Application de  [tx-robot] updated from transifex
Application de  Bump sass from 1.50.0 to 1.52.1
Application de  [tx-robot] updated from transifex
Application de  Update psalm baseline
Application de  [tx-robot] updated from transifex
Application de  Rename user-agent used by the Webcal Refresh Service
Application de  Fix psalm issues related to the user backend
Utilisation de l'information de l'index pour reconstruire un arbre de base...
M       build/psalm-baseline.xml
Retour à un patch de la base et fusion à 3 points...
Fusion automatique de build/psalm-baseline.xml
Application de  Update DAV availability vue component to standard
Application de  [tx-robot] updated from transifex
Application de  Adjust docs to the same as the implementation
Application de  Add API to change preferences
Application de  Revert api break
Application de  Remove at() matcher for tests for files app
Application de  Fix various errors reported by eslint
Application de  Improve warning about missing pnctl
Application de  Fix return type and make type stricter
Application de  Modernize the dav admin settings
Application de  Require a recent phpunit version in integration and acceptance
Application de  [tx-robot] updated from transifex
Application de  Fix handling events without names in activities
Application de  Also trim the name
Application de  Remove at() matcher use from encryption tests
Application de  Remove at() matcher in files_sharing tests
Application de  [tx-robot] updated from transifex
Application de  [tx-robot] updated from transifex
Application de  Fix missing status with predefined status in drop-down
Application de  Don't overwrite certain values if the refreshing comes in via short updates
Application de  Fix unit tests
Application de  Move Gd failed operations to debug level
Application de  Add lint job for PHP8.2
Application de  allow use of rememberme cookies for IApacheAuth backends
Application de  Introduce event for loading additional script on projects
Application de  Move files_sharing over to new event for projects
Application de  Fix loading groupfolder info when share api is disabled
Application de  [tx-robot] updated from transifex
Application de  [tx-robot] updated from transifex
Application de  Handle non existing settings again
Application de  Update core to PHP 7.4 standard
Application de  Update psalm baseline
Application de  Fix node tests ci
Application de  Fix docblock for return type
Application de  Bump core-js from 3.22.5 to 3.22.7
Application de  Add a method to get estimated export size in IMigrator
Application de  Implement getExportEstimatedSize in migrators
Application de  Move new IMigrator method to a specific interface ISizeEstimationMigrator
Application de  Add the ISizeEstimationMigrator interface for method getExportEstimatedSize
Application de  Update method name
Application de  Skip avatar on failure
Application de  Update contacts estimation
Application de  Update calendar estimation
Application de  [tx-robot] updated from transifex
Application de  Properly calculate primary element based on background luminance
Application de  Cleanup tempfiles after each cron job run
Application de  Use JSON_THROW_ON_ERROR instead of custom error handling
Application de  Fix Nextcloud Android app not showing proper name
Application de  Improve JSConfigHelper code quality a bit
Application de  Restore old behaviour of sending flase for not found apps
Application de  Fix unit tests
Application de  Update psalm baseline
Application de  Use SettingsSection in workflowengine settings
Application de  Update 3rdparty for symfony/translation:4.4.41
Utilisation de l'information de l'index pour reconstruire un arbre de base...
A       3rdparty
Retour à un patch de la base et fusion à 3 points...
CONFLIT (modification/suppression) : 3rdparty supprimé dans HEAD et modifié dans Update 3rdparty for symfony/translation:4.4.41. Version Update 3rdparty for symfony/translation:4.4.41 de 3rdparty laissée dans l'arbre.
error: Échec d'intégration des modifications.
l'application de la rustine a échoué à 0076 Update 3rdparty for symfony/translation:4.4.41
astuce: Utilisez 'git am --show-current-patch' pour visualiser le patch en échec
Resolve all conflicts manually, mark them as resolved with
"git add/rm <conflicted_files>", then run "git rebase --continue".
You can instead skip this commit: run "git rebase --skip".
To abort and get back to the state before "git rebase", run "git rebase --abort".
jh-arawa@administrateur-ThinkBook-15-G2-ITL:~/Documents/GitHub/server$ 
jh-arawa@administrateur-ThinkBook-15-G2-ITL:~/Documents/GitHub/server$ 
jh-arawa@administrateur-ThinkBook-15-G2-ITL:~/Documents/GitHub/server$ git status 
rebasage en cours ; sur 766a30e9619
Vous êtes en train de rebaser la branche 'patch-11' sur '766a30e9619'.
  (réglez les conflits puis lancez "git rebase --continue")
  (utilisez "git rebase --skip" pour sauter ce patch)
  (utilisez "git rebase --abort" pour extraire la branche d'origine)

Chemins non fusionnés :
  (utilisez "git restore --staged <fichier>..." pour désindexer)
  (utilisez "git add/rm <fichier>..." si nécessaire pour marquer comme résolu)
        supprimé par nous :       3rdparty

aucune modification n'a été ajoutée à la validation (utilisez "git add" ou "git commit -a")
```

Thanks by advance for your help.

**Comment by szaimen** [comment]
All right! If it does not work, creating a new branch and a new PR is probably easiert going forward...

**Pull request closed without merging**